src/pulp_docs/cli.py

```diff
@@ -43,10 +57,24 @@ def draft_callback(ctx: click.Context, param: click.Parameter, value: bool) -> b
     help="Don't fail if repositories are missing.",
 )
 
+paths_option = click.option(
+    "--paths",
+    envvar="PULPDOCS_PATHS",
+    expose_value=False,
+    default="",
+    callback=find_paths_callback,
+    help="A semicolon separated list of repository paths. Accept glob patterns.",
+)
```

Comment:
In order to keep it close to similar existing PATH variables can we agree to do this?
```suggestion
path_option = click.option(
    "--path",
    envvar="PULPDOCS_PATH",
    expose_value=False,
    default="",
    callback=find_path_callback,
    help="A colon separated list of repository paths. Accepts glob patterns.",
)
```